Application/src/view/visuals/component/Message.java

```diff
@@ -231,11 +237,27 @@ public void setStatic(boolean staticIndicator)  {
     public void renderDefault(GraphicsContext gc) {
 
 
+        if(trails.size() != 0) {
+            Trail first;
+            //Draws all trails except the first and last one
+            for (int i = 1; i < trails.size() - 1; i++) {
+                // Points to the first index of the trail array
+                first = trails.get(i);
+                gc.drawImage(trail, first.getXcoordinate(), (first.getYcoordinate() + 18), first.getWidth(), first.getHeight());
```

Comment:
How come it says this is still changed? It looks the same as before, just wothout the "for (Trail t: trails)" ?

Reply:
nvm, my comment some how disapeared.  Pls change back to "gc.drawImage(trail, t.getXcoordinate(), (t.getYcoordinate() + 18), t.getWidth(), t.getHeight());"

Reply:
@IsabelleTornqvist that is how i fixed the bug, notice the "trails.size() - 1" which is looping through all indexes except the last one so we can add the trail ;)

Reply:
@IsabelleTornqvist your comment disappear if i push changes, click on "show outdated" to see history

Reply:
Might be the start index of 1?

Reply:
@Sebastian-Fransson what do you mean?